roles/matrix-ldap-registration-proxy/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- ansible.builtin.import_tasks: "{{ role_path }}/tasks/init.yml"
+  tags:
+    - always
+
+- ansible.builtin.import_tasks: "{{ role_path }}/tasks/validate_config.yml"
```

Comment:
`validate_config.yml` being empty (as it is now) will cause some Ansible error when importing.

In another review comment, I'm advising that you start making use of this file though (adding stuff to `validate_config.yml`), so it will no longer be a problem.